src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVariantContextUtils.java

```diff
@@ -1349,4 +1351,119 @@ private static int indexOfSameAllele(final VariantContext vc, final Allele allel
 
         return -1;
     }
+
+    /**
+     * Add chromosome Counds (AC, AN and AF) to the VCF header lines
```

Comment:
So there are.  Wait on doing those renames until I talk with a few people here about what the right name is.  I don't want it to be inconsistent, but I think it's confusing how it is at the moment.
1.  Also, I noticed that VariantContextUtils has `calculateChromosomeCounts` which seems to be very similar to the new methods in this pr.  Did we know about those going in to these changes?   Are we duplicating code that already exists?  
2.  I also noticed that ChromosomeCountConstants seems to be totally redundant with ChromosomeCounts.  Could you remove ChromoSomeCountConstants while you're in this code?  It looks like add ChromomeCounts should probably just addAll ChromosomeCounts.getDescriptions as well..